cmd/influxd/upgrade/upgrade.go

```diff
@@ -149,6 +175,13 @@ func NewCommand() *cobra.Command {
 			Desc:    "path for boltdb database",
 			Short:   'm',
 		},
+		{
+			DestP:   &options.target.configsPath,
+			Flag:    "configs-path",
```

Comment:
Could we rename the flag to be even more specific, i.e. `influx-configs-path`? I'm worried about confusion with the existing `config-file` argument.